### DISCUSSION TITLE
FaceSwapLab - WARNING - skip, similarity to low, sim

### DISCUSSION THREAD

**Discussion by robertinosro** [glucauze/sd-webui-faceswaplab#174] [Q&A]
So i'm getting this error:
 FaceSwapLab - WARNING - skip, similarity to low, sim = {0: 0.13658765} (target 0.21) ref sim = {0: 0.13658765} (target = 0.19)
I tried a lot of settings but it seems i can't get it to work. Images are square, good quality
Check similarity On
Compute similarity ON
I'm kinda noob in this, any help would be appreciated
Thank you

**Comment by a3nima** [comment]
Why you need similarity ? You can choose for yourself which face you want to swap?

**Reply by robertinosro** [reply]
i need a specific face to be swapped into generated ai image. That's why

**Comment by a3nima** [comment]
I'm not sure, that feature to me suggests more to be a function to automatically choose a face, that fits best. If you are creating a new photo you can use prompting to get a similar person. But, as I've not used that function, I'm probably not the right person to help you here :D I can offer you help, to achieve your goal, maybe, if you post your whole process and what you want to achieve. For example, what happens if you dont use the feature?
